test/bitmap-constructor-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bitmap = require('../model/bitmap-constructor.js');
+const readFileHelper = require('../lib/bitmap-file-helper.js');
+const fs = require('fs');
+const Buffer = require('buffer').Buffer;
+
+describe('Testing constructor', function() {
+  describe('Constructor function', function() {
+    before('Build bitmap object', function() {
+      this.bitmapObj = new bitmap.BufferData(fs.readFileSync(readFileHelper.filePath));
+    });
+    it('should construct an object', function() {
+      expect(this.bitmapObj).to.be.an('object');
+    });
+    it('should have colorArray property', function() {
+      expect(this.bitmapObj).to.have.property('colorArray').to.have.length.within(54, 1078);
+      expect(this.bitmapObj).to.have.property('colorArray').to.not.have.length(0);
+    });
+  });
+  describe('Transform methods', function() {
+    before('Build bitmap object', function() {
```

Comment:
@brittdawn @jonathanheemstra @peterkim2 great use of the `before` hook to create new bitmap data